config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  devise_for :users
+  resources :entities
+  resources :categories
+  resources :users
```

Comment:
- [ ] Kindly only add routes that a being used by the app. We can do something like so: `resource :entities, only: [create, show, ....]`